styles.css

```diff
@@ -1,3 +1,226 @@
 body {
-  background-color: white;
+  color: #2f3f30;
+  margin: 0px;
+  width: 100%;
+}
+
+html {
+  scroll-behavior: smooth;
+  background-color: #b0d9b1;
+}
+
+a {
+  text-decoration: none;
+  color: #2f3f30;
+}
+
+h1 {
+  margin: auto;
+}
+
+h2 {
+  padding: 20px;
+  margin: 0px;
+}
+
+h3 {
+  padding: 0px 10px;
+}
+
+h4 {
+  text-align: center;
+}
+
+#head {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #d0e7d2;
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+  padding: 3px 10px;
+  scroll-behavior: smooth;
+}
+
+@keyframes photo-expand {
+  from {
+    height: 100px;
+  }
+  to {
+    height: 300px;
+  }
+}
+
+.photo-img {
+  border-radius: 100%;
+  margin-right: 20px;
+}
+
+.photo-img:hover {
+  animation-name: photo-expand;
+  animation-duration: 4s;
+  height: 300px;
+  z-index: 1;
+}
+
+.navi {
+  height: 50px;
+  display: flex;
+  justify-content: flex-start;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #79ac78;
+
+  border-bottom: 3px solid #618264;
+  border-top: 3px solid #618264;
+}
+
+.navi-link {
+  border-right: 2px solid #618264;
+}
+
+.navi-link:hover {
+  background-color: #567855;
+}
+
+.navi-link:active {
+  background-color: #506b52;
+}
+
+.content {
+  scroll-behavior: smooth;
+  width: 200%;
+  display: flex;
+  justify-content: space-between;
+  overflow-x: hidden;
+  max-width: 100%;
+}
+
+#about-me {
+  position: relative;
+  min-width: 100%;
+  max-width: 100%;
+  width: 100%;
+  height: 600px;
+  background-color: #b0d9b1;
+}
+
+#project {
+  position: relative;
+  min-width: 100%;
+  max-width: 100%;
+  background-color: #b0d9b1;
+  width: 100%;
+  height: 600px;
+}
+
+#contact {
+  width: 100%;
+  height: 100px;
+  background-color: #79ac78;
+  padding-bottom: 5px;
+}
+
+#info-photo {
+  display: flex;
+  justify-content: center;
+}
+
+#info {
+  padding: 50px;
+  border-radius: 10% 0 10% 0;
+  box-shadow: 5px 5px #4d6d4c;
+  background-color: #79ac78;
+  margin: 20px;
+  height: 350px;
+  width: 600px;
+}
+
+#about-me-photo {
+  margin-top: 30px;
+  border-radius: 10% 0 10% 0;
+  height: 400px;
+  box-shadow: 5px 5px #4d6d4c;
+}
+
+#info-list {
+  display: flex;
+  justify-content: space-between;
+}
+
+#project-container {
```

Comment:
just wondering, why sometimes you choose to use ids, sometimes classes? Any specific reason?

Reply:
I use id for some specific elements. I use class for some reusable elements,  in case there will be similar elements will be added for the future use.